packages/auth-core/src/core.ts

```diff
@@ -138,17 +138,19 @@ export class Auth {
     return this.getToken(code, verifier);
   }
 
-  async resendVerificationEmail(verificationToken: string) {
+  async resendVerificationEmail(verificationToken: string, verifyUrl: string) {
```

Comment:
This is shared also in express.
Now I wonder if the verifyUrl should be optional? @scotttrinh 

Or add it everywhere where it fails?

Reply:
We don't pass `verifyUrl` when you have a `verificationToken` since the URL is contained within that signed token, so I think you can just remove it for this function.